src/main/java/seedu/duke/controllers/OrderController.java

```diff
@@ -71,9 +71,10 @@ private void getAllOrderPrice() {
         System.out.printf("Total value of all orders: %f. \n", orderManager.getAllOrderValue());
     }
 
-    private void printReceipt() {
-        System.out.println("These are all your orders receipts. \n");
-        orderManager.printReceipt();
+    private void printOrder() throws OperationTerminationException {
+        int userInputInt = InputParser.getInteger("Enter the order you want to display: ");
+        System.out.println("These is your order. \n");
+        orderManager.getOrder(userInputInt).printOrder();
```

Comment:
what if the integer is indexoutofbounds? discuss with ziyan

Reply:
just a recommendation: you can create a findOrder() method or smth along those lines that let's u find if an order is in the orders array

Reply:
ok

Reply:
instead of making this method throw indexoutofboundsexception, you should catch it and then tell the user if it is an invalid index.